packages/types/src/sszTypes.ts

```diff
@@ -21,6 +21,7 @@ const typesByFork = {
   [ForkName.capella]: {...phase0, ...altair, ...bellatrix, ...capella},
   [ForkName.deneb]: {...phase0, ...altair, ...bellatrix, ...capella, ...deneb},
   [ForkName.electra]: {...phase0, ...altair, ...bellatrix, ...capella, ...deneb, ...electra},
+  [ForkName.fulu]: {...phase0, ...altair, ...bellatrix, ...capella, ...deneb, ...electra},
```

Comment:
could potentially already do this

```suggestion
  [ForkName.fulu]: {...phase0, ...altair, ...bellatrix, ...capella, ...deneb, ...electra, ...fulu},
```

Reply:
I'm not sure if there will be conflicts having two of the same type so i left it out...

Reply:
I only added the `electra as fulu` so that the index of `ssz` didnt throw and not necessarily to proxy those types

Reply:
> I'm not sure if there will be conflicts having two of the same type so i left it out...

no, this should work fine, as those should override forks of the earlier forks, that's the whole purpose of copying those like this here

Reply:
fine with leaving this out for now though, once we add new type we will notice it

Reply:
note to self: don't do it